OpenSim/Tools/InverseKinematicsTool.cpp

```diff
@@ -332,10 +332,17 @@ bool InverseKinematicsTool::run()
             ikSolver.track(s);
             
             if(_reportErrors){
-                Array<double> markerErrors(0.0, 3);
+                Array_<Vec3> markerRefValues(nm);
+                Array<double> markerErrors(0.0, nm);
                 double totalSquaredMarkerError = 0.0;
                 double maxSquaredMarkerError = 0.0;
                 int worst = -1;
+                int ignore_cnt = 0;
+
+                ikSolver.getMarkerReferenceValues(markerRefValues);
+                for (const Vec3& val : markerRefValues) {
+                    val.isNaN() ? ignore_cnt++ : 0;
```

Comment:
I think it makes more sense to write this as `if (val.isNaN()) ++ignore_cnt;` because the value of the `? :` expression is not used.

Reply:
Does this count skip over references for markers that do not have a corresponding model marker?

Reply:
> Does this count skip over references for markers that do not have a corresponding model marker? 

Markers without a corresponding model marker are not part of the reference values returned because they are not added to the solver to begin with.

Reply:
Okay thanks for clarifying.